assets/js/atomic/blocks/product-elements/rating/style.scss

```diff
@@ -56,6 +56,37 @@
 		margin-top: 0;
 		margin-bottom: $gap-small;
 	}
+
+	&__norating-container {
+		display: inline-flex;
+		flex-direction: row;
+		align-items: center;
+		gap: $gap-smaller;
+	}
+
+	&__norating {
+		display: inline-block;
+		overflow: hidden;
+		position: relative;
+		width: 1em;
+		height: 1.618em;
+		line-height: 1.618;
+		font-size: 1em;
+		/* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+		font-family: star;
+		font-weight: 400;
+		-webkit-text-stroke: 2px var(--wp--preset--color--contrast);
+
+		&::before {
+			content: "\53";
+			top: 0;
+			left: 0;
+			right: 0;
+			position: absolute;
+			color: transparent;
```

Comment:
So the star wasn't showing for me is because of this line. I think you probably want to use opacity?

Reply:
Which browser are you using?

Reply:
Chrome.

Reply:
I tested with Firefox and Chrome and I'm not able to replicate it:

| Firefox | Chrome |
|--------|--------|
|![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/66ebbc0f-689e-4e85-96ae-12e888bebbd2)|![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/97563901-44d8-4c9f-b11d-3f29700ec26d)|